Switch-valuesChallenge/js-inputs-and-strings_switch-values/js/index.js

```diff
@@ -14,13 +14,24 @@ Hint: the HTML entity for the switch button is: &updownarrow
 */
 
 const firstInput = document.querySelector('[data-js="first-input"]');
+const secondInput = document.querySelector('[data-js="second-input"]');
+
 const uppercaseButton = document.querySelector('[data-js="button-uppercase"]');
 const lowercaseButton = document.querySelector('[data-js="button-lowercase"]');
 
+const switchButton = document.querySelector('[data-js="button-check"]');
+
 uppercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toUpperCase();
+  secondInput.value = secondInput.value.toUpperCase();
 });
 
 lowercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toLowerCase();
+  secondInput.value = secondInput.value.toLowerCase();
+});
+
+switchButton.addEventListener("click", (e) => {
```

Comment:
Switch button isn't working. You are only turning first input to lowercase, which was not the task, here. This is what you need in order to switch the two inputs:
const firstValue = firstInput.value;
firstInput.value = secondInput.value;
secondInput.value = firstValue;